src/test/java/net/obvj/performetrics/PerformetricsTest.java

```diff
@@ -264,4 +263,12 @@ void setDefaultPrintStyleForDetails_null_doesNotUpdateConfiguration()
         }
     }
 
+    @Test
+    void monitorOperationDoesNotReturnNullValue()
+    {
+        MonitoredRunnable monitored = Performetrics.monitorOperation(
+                () -> {throw new RuntimeException();}, WALL_CLOCK_TIME, CPU_TIME);
+        assertThat(monitored, notNullValue());
```

Comment:
Also, it would be better to test that elapsedTime brings a positive value. For example:

````java
assertThat(monitored.elapsedTime(Type.WALL_CLOCK_TIME, TimeUnit.NANOSECONDS), isPositive());
````

Reply:
Not needed anymore since the change will be in the Javadoc